dll/win32/advapi32/sec/lsa.c

```diff
@@ -692,6 +692,12 @@ LsaEnumerateTrustedDomainsEx(IN LSA_HANDLE PolicyHandle,
     return Status;
 }
 
+static
+LPVOID
+LsaAllocMemory(SIZE_T cb)
+{
+    return RtlAllocateHeap(RtlGetProcessHeap(), 0, cb);
```

Comment:
In view of my other comment, I think the better fix here would be to fix the LsaFreeMemory implementation so that it calls MIDL_user_free instead.